### PR TITLE
[CI] Install conan using pkg manager instead of pip

### PR DESCRIPTION
**Pull request by AnotherFoxGuy**
This should install Conan a little faster than pip

<details>
<summary>Contributor stuff</summary>

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [x] I made sure the code compiles on my machine
- [ ] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

</details>

### CODE REVIEW

**Comment by nbsp** [comment]
Please do not replace the contribution checkboxes with emoji.

File: scripts/ci/environment.sh [hunk 1]
```diff
@@ -21,4 +21,4 @@ gh_export GIT_HASH_SHORT="$(git show -s --format='%h')"
 gh_export AUDACITY_BUILD_TYPE="RelWithDebInfo"
 gh_export AUDACITY_INSTALL_PREFIX="${repository_root}/build/install"
 
-gh_export GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+gh_export GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
```
Comment (nbsp) [inline scripts/ci/environment.sh]: I assume that's your editor doing this automatically. Please install an EditorConfig plugin.

**Review by nbsp — CHANGES_REQUESTED** [review]

**Review by nbsp — DISMISSED** [review]

File: scripts/ci/dependencies.sh [hunk 2]
```diff
@@ -6,11 +6,12 @@ set -euxo pipefail
 
 if [[ "${OSTYPE}" == msys* ]]; then # Windows
 
-    # Python packages
-    pip_packages=(
-        conan
-    )
-    pip3 install "${pip_packages[@]}"
+    if which choco; then
+        choco install sccache conan -y
```
Comment (fossdd) [inline scripts/ci/dependencies.sh]: Please use shell array for package listing like `pip_packages`

File: scripts/ci/dependencies.sh [hunk 3]
```diff
@@ -51,14 +51,10 @@ else # Linux & others
...
-    pip_packages=(
-        conan
-    )
-
-    which cmake || pip_packages+=( cmake ) # get latest CMake when inside Docker image
-
-    pip3 install wheel setuptools # need these first to install other packages (e.g. conan)
-    pip3 install "${pip_packages[@]}"
+    # Conan 
+    if which dpkg; then
```
Comment (fossdd) [inline scripts/ci/dependencies.sh]: Please put these commands inside "if which apt-get". ' If apt-get is installed dpkg is also. Please also try to use curl and pipe the output into dpkg that no temp files are required.

**Review by fossdd — CHANGES_REQUESTED** [review]

File: scripts/ci/dependencies.sh (hunk 3 above)
Comment (AnotherFoxGuy) [reply scripts/ci/dependencies.sh]: > Please also try to use curl and pipe the output into dpkg that no temp files are required.

Is this something you can do? Looking online it seems that dpkg only can read files from disk

Comment (fossdd) [reply scripts/ci/dependencies.sh]: Mh. Okay, sorry. I also don't get it working. Looks like the devs don't implemented it.. 🙄

**Review by fossdd — DISMISSED** [review]

**Review by nbsp — CHANGES_REQUESTED** [review]
With the emoji prepended, the commit message is now 68 characters long. Please refrain from using emoji in your commit messages -- you're the only one who uses them here, and they don't work outside of GitHub.

**Comment by nbsp** [comment]
```
From: Edgar <Edgar@AnotherFoxGuy.com>
Date: Sat, 24 Jul 2021 20:27:54 +0200
Subject: [PATCH] =?UTF-8?q?=F0=9F=91=B7=20Install=20conan=20using=20pkg=20?=
 =?UTF-8?q?manager=20instead=20of=20pip?=
```

Yeah, emoji don't work.

**Comment by AnotherFoxGuy** [comment]
> Yeah, emoji don't work.

That seems more like an issue with sourcehut, since I've had plenty of emails with emoji in the subject line

**Comment by nbsp** [comment]
> > Yeah, emoji don't work.
> 
> That seems more like an issue with sourcehut, since I've had plenty of emails with emoji in the subject line

This isn't sourcehut: this is the `.patch` file generated *by GitHub* for use in git.
I think the problem is that git doesn't handle UTF-8 and uses US-ASCII instead.

**Comment by nbsp** [comment]
I'm not surprised by this at all: GitHub loves to ignore industry standards and lock projects to their ecosystem. 
For example, `code blocks` in commit messages are only rendered correctly in GitHub.

**Comment by Be-ing** [comment]
I agree with not using emojis in commit messages.

**Comment by AnotherFoxGuy** [comment]
Well, CONTRIBUTING.md does not specify what you can and can't put in the commit messages ¯\\_(ツ)_/¯

**Comment by nbsp** [comment]
The commit message is just a bit too long. Try this:

```
ci: Install conan using pkg manager instead of pip
```

**Review by nbsp — APPROVED** [review]

**Review by fossdd — APPROVED** [review]

**Pull request merged**